File: works/1475.py
nums = [0 for _ in range(10)]
number = input()
for i in range(len(number)):
    nums[int(number[i])]+=1

a = 0; b = 0
for i in range(10):
    if i == 6 or i == 9: continue
    a = max(a, nums[i])
b = nums[6] + nums[9]
if b % 2 != 0: b = b // 2 + 1
else: b = b // 2
print(max(a,b))

File: works/1475.cpp
#include <iostream>
#include <string>
#include <algorithm>
using namespace std;

int nums[10];

int main()
{
    string number; cin >> number;
    for(int i = 0; i < number.size(); i++) nums[number[i]-'0']++;
    int a = 0, b = 0;
    for(int i = 0; i < 10; i++)
    {
        if(i == 6 || i == 9) continue;
        a = max(a, nums[i]);
    }
    b = nums[6] + nums[9];
    if(b % 2 != 0)  b = b / 2 + 1;
    else b = b / 2;
    cout << max(a, b);
}